eventMacro.pl

```diff
@@ -92,10 +92,13 @@ sub commandHandler {
 		message 
 			"eventMacro MACRO: run macro MACRO\n".
 			"eventMacro list: list available macros\n".
-			"eventMacro status: shows current status\n".
-			"eventMacro stop: stop current macro\n".
-			"eventMacro pause: interrupt current macro\n".
-			"eventMacro resume: resume interrupted macro\n".
+			"eventMacro macro_status: shows current macro status\n".
```

Comment:
These names are too long to type. It's fine to assume people have an IDE and will be able to autocomplete long variable names, but these are going to be typed by end users. They should be short and easy to remember. I vote to keep the original names for the `macro` set (ie, `macro status`, `macro stop`, `macro pause`, `macro resume`).